### PR TITLE
MNT: Do not write config on import

### PR DESCRIPTION
**Pull request by pllim**
This functionality will be deprecated in `astropy` in a future release. See astropy/astropy#11497 and https://groups.google.com/g/astropy-dev/c/KVzPgeihxO8

cc @saimn

### CODE REVIEW

**Comment by saimn** [comment]
The config file that was created by default should be replaced by the automatically generated config in the docs: https://github.com/astropy/sphinx-astropy/blob/main/sphinx_astropy/ext/generate_config.py

**Comment by pllim** [comment]
@saimn -- Thanks for the review. Done!

File: astroquery/_astropy_init.py [hunk 1]
```diff
@@ -25,36 +25,9 @@
 
 if not _ASTROPY_SETUP_:  # noqa
```
Comment (saimn) [inline astroquery/_astropy_init.py:26]: Seems a good opportunity to remove `_ASTROPY_SETUP_` as well.

Comment (pllim) [reply astroquery/_astropy_init.py:26]: I am not sure. `astroquery` is still using `astropy-helpers`.

Comment (saimn) [reply astroquery/_astropy_init.py:26]: Oh, right  :astonished:

Comment (bsipocz) [reply astroquery/_astropy_init.py:26]: Yep, there were bigger fishes to fry than fixing what is not yet broken, but hopefully, we will have some capacity in the future to do some refactoring.

**Review by bsipocz — APPROVED** [review]
All looks good! Thanks @pllim!

**Pull request merged**